### ISSUE TITLE
Android: some spacing is added when dragging

### ISSUE THREAD

**Issue by antoinerousseau** [deanmcpherson/react-native-sortable-listview#18]
I don't have any animation in Android.
Also, why is there some spacing added below the dragged row when I start dragging?

Using Genymotion, react-native 0.27.0, react 15.1.0, react-native-sortable-listview 0.0.7.

**Comment by sudosiyang** [comment]
// Enable LayoutAnimation under Android
```
    if (Platform.OS === 'android') {
      UIManager.setLayoutAnimationEnabledExperimental(true)
    }
```

**Comment by antoinerousseau** [comment]
thanks @sudosiyang!

**Comment by antoinerousseau** [comment]
I still have a spacing issue though.

**Comment by deanmcpherson** [comment]
Hey @antoinerousseau, can you elaborate on what exactly the spacing issue you're experiencing?

**Comment by antoinerousseau** [comment]
@deanmcpherson I was going to make you a screencast, but I updated react-native to 0.28 in between and my problem seems to have disappeared!
Basically, when I started dragging (so like one second after starting pressing the element) an empty space of the same height than the element was created just below it.

**Comment by deanmcpherson** [comment]
Hey @antoinerousseau, I think this is resolved now from 0.0.8.
